### DISCUSSION TITLE
Add the Homebrew installation to Navidrome documentation and maintain this installation method.

### DISCUSSION THREAD

**Discussion by Intro1997** [Homebrew/discussions#5645] [Everyday usage] [answered]
### Output of `brew config`

```shell
None
```

### Output of `brew doctor`

```shell
None
```

### Description of issue

Navidrome is a great tool, but the [release page](https://github.com/navidrome/navidrome/releases/tag/v0.53.3) of the repo does not provide an arm64 version for macOS. I [learned from the maintainer of navidrome](https://github.com/navidrome/navidrome/discussions/3318#discussioncomment-10768519) that it can be installed using homebrew. I have used it for a while and it works well without any problems.

However, this installation method is not mentioned in the documentation of navidrome because it is not packaged by the maintainer of navidrome(see [here](https://github.com/navidrome/navidrome/discussions/3318#discussioncomment-10784806)). The maintainer of navidrome welcomes me to submit a document update for the supplementary installation method and hopes to explain where this installation method is maintained. 

Installing and using navidrome through homebrew facilitates subsequent updates and management, so I hope that this installation method can be updated to the documentation of navidrome.

I noticed by checking [blame](https://github.com/Homebrew/homebrew-core/blame/master/Formula/n/navidrome.rb) that the maintainers of homebrew navidrome are @printfn and @chenrui333. I would like to ask if you agree to update this installation method to the official documentation?

**Comment by SMillerDev** [comment, marked as answer]
Homebrew doesn't have individual package maintainers. If you want to add something somewhere, go ahead.

**Reply by Intro1997** [reply]
> Homebrew doesn't have individual package maintainers. If you want to add something somewhere, go ahead.

Sorry, I don't quite understand some of Homebrew's rules about contributors. Are you saying that the author of *.rb formula file that contributes to homebrew has no responsibility to maintain it?

**Reply by SMillerDev** [reply]
> Are you saying that the author of *.rb formula file that contributes to homebrew has no responsibility to maintain it?

No explicit responsibility, no. If it breaks and nobody is there to fix it it'll just be disabled and removed.

**Reply by Intro1997** [reply]
> > Are you saying that the author of *.rb formula file that contributes to homebrew has no responsibility to maintain it?
> 
> No explicit responsibility, no. If it breaks and nobody is there to fix it it'll just be disabled and removed.

Ok, I got it. Thanks for your reply.
